pages/common/git-show-index.md

```diff
@@ -0,0 +1,12 @@
+# git show-index
+
+> Show the packed archive index.
+> More information: <https://git-scm.com/docs/git-show-index>.
+
+- Read the `.idx` file for a Git packfile and dump its contents to stdout:
```

Comment:
```suggestion
- Read an `.idx` file for a Git packfile and dump its contents to stdout:
```

Reply:
This is a bit tricky.  @CleanMachine1 I (and I believe most people although that's speculation) would pronouce the file extension `.idx` as "dot I D X", hence the article there would be `a`, not `an`, because there's no literal or phonetic vowel there.

Reply:
yea, didn't think this one through, the `the` should be out of the question since it makes .idx the noun rather than file

Reply:
```suggestion
- Read a `.idx` file for a Git packfile and dump its contents to stdout:
```